Musigwa/SendIT-API
# #161747706 change destination endpoints

## What does this PR do (?)

It makes the user able to change the destination of the specific parcel order with the given id.

## Description of Task to be completed (?)

The following Route should be working:

- PUT /parcels/<parcel_id>/destination

## How should this be manually tested (?)

After cloning this repo and switch to this branch, run the dependencies installation with either npm or yarn depending on what package manager you're using,

- Using any RESTful Client tool (Usually the postman), provide the target parcel_id, set the new destination within the request body. 

Key: Make sure to spell the destination correctly. It must be in lowercase like so: **destination**

Key: Content-Type value: application/json

## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. 161747706
